src/components/Pill/PillColors.ts

```diff
@@ -0,0 +1,28 @@
+export const PillColors = {
```

Comment:
I think this should go to enums, this file is not exported so nobody can use it outside DS.

Reply:
Isn't that exactly what we want? We don't want the color range in the theme so it wouldn't be spread all around outside the pills - we just want to offer enums to our consumers.

Reply:
We don't want this to be in theme, because we don't want to encourage people to use these colours for styling. But they need it to be able to build a tag color selector so it needs to be exported anyway